OBJC_INTEROP.md

```diff
@@ -0,0 +1,124 @@
+# _Kotlin/Native_ interoperability with Swift/Objective-C
+
+This documents covers some details of Kotlin/Native interoperability with
+Swift/Objective-C.
+
+## Usage
+
+Kotlin/Native provides bidirectional interoperability with Objective-C.
+Objective-C frameworks and libraries can be used in Kotlin code if
+properly attached to the build (system frameworks are attached by default).
+See e.g. "Interop libraries" in
+[Gradle plugin documentation](GRADLE_PLUGIN.md#building-artifacts).
+Swift library can be used in Kotlin code if its API is exported to Objective-C
+with `@objc`.
```

Comment:
Maybe explicitly mention "Pure Swift modules are not yet supported"?

Reply:
Instead of the last sentence or after it?

Reply:
After